spyderlib/config.py

```diff
@@ -484,6 +485,7 @@ def is_ubuntu():
               '_/save current layout': "Shift+Alt+S",
               '_/toggle default layout': "Shift+Alt+Home",
               '_/layout preferences': "Shift+Alt+P",
+              '_/show toolbars': "Alt+Shift+T",

```

Comment:
@goanpeca, can you confirm that this working for you? I remember you told there was some problem with this change.